src/sqlfluff/dialects/dialect_redshift.py

```diff
@@ -323,13 +398,16 @@ class StatementSegment(BaseSegment):
 
     parse_grammar = redshift_dialect.get_segment("StatementSegment").parse_grammar.copy(
         insert=[
-            Ref("TableAttributeSegment"),
+            Ref("AlterGroupSegment"),
+            Ref("AlterUserSegment"),
             Ref("ColumnAttributeSegment"),
             Ref("ColumnEncodingSegment"),
-            Ref("CreateUserSegment"),
+            Ref("CreateExternalTableAsStatementSegment"),
+            Ref("CreateExternalTableStatementSegment"),
             Ref("CreateGroupSegment"),
-            Ref("AlterUserSegment"),
-            Ref("AlterGroupSegment"),
+            Ref("CreateUserSegment"),
+            Ref("PartitionedBySegment"),
+            Ref("TableAttributeSegment"),
```

Comment:
Again you shouldn't be reordering things that are unrelated to your PR, it just makes the diffs unclear...

Reply:
Makes sense, reverted the ordering